src/Components/Results/Results.js

```diff
@@ -339,38 +341,21 @@ const Results = () => {
   };
 
   const CustomGridTreeDataGroupingCell = (props) => {
-    const { id, field, rowNode } = props;
+    const { id, rowNode } = props;
     const apiRef = useGridApiContext();
     const filteredDescendantCountLookup = useGridSelector(apiRef, gridFilteredDescendantCountLookupSelector);
     const [navListOpen, setNavListOpen] = useState(false);
     const openNavList = () => {
       setNavListOpen(!navListOpen);
     };
 
-    const handleKeyDown = (event) => {
-      if (event.key === ' ') {
-        event.stopPropagation();
-      }
-      if (isNavigationKey(event.key) && !event.shiftKey) {
-        apiRef.current.publishEvent('cellNavigationKeyDown', props, event);
-      }
-    };
-
-    const handleClick = (event) => {
-      apiRef.current.setRowChildrenExpansion(id, !rowNode.childrenExpanded);
-      apiRef.current.setCellFocus(id, field);
-      event.stopPropagation();
-    };
-
```

Comment:
I moved these functions inside the `CustomDetailPanelToggle` component since `IconButton` will need them to handle these events accordingly.